### ISSUE TITLE
pcluster configure command failure when the account has a no IPv4 CIDR subnet.

### ISSUE THREAD

**Issue by k11o** [aws/aws-parallelcluster#4816]
**Required Info:**
 - AWS ParallelCluster version [e.g. 3.1.1]:3.4.1
 - Full cluster configuration without any credentials or personal data.: no configuration data
 - Cluster name: N/A
 - Output of `pcluster describe-cluster` command.: N/A

**Bug description and how to reproduce:**
A clear and concise description of what the bug is and the steps to reproduce the behavior.
1. create IPv6 CIDR Only subnet in your VPC.
2. run ```pcluster configure``` command.
3. the command ends suddenly after ```Maximum instance count [10]:```

**Additional context:**
The error was reported in ~/.parallelcluster/pcluster-cli.log
```
2023-01-23 20:20:06,054 - INFO - common.py:134:_log_boto3_calls() - Executing boto3 call: region=ap-northeast-1, service=ec2, operation=DescribeInstanceTypes, params={'InstanceTypes': ['t2.micro']}
2023-01-23 20:20:07,484 - ERROR - entrypoint.py:256:main() - Unexpected error of type AttributeError: 'NoneType' object has no attribute 'split'
Traceback (most recent call last):
  File "/mnt/c/Users/t_kono/Develop/pcluster-test/env/lib/python3.9/site-packages/pcluster/cli/entrypoint.py", line 234, in main
    ret = run(sys.argv[1:])
  File "/mnt/c/Users/t_kono/Develop/pcluster-test/env/lib/python3.9/site-packages/pcluster/cli/entrypoint.py", line 228, in run
    return _run_operation(model, args, extra_args)
  File "/mnt/c/Users/t_kono/Develop/pcluster-test/env/lib/python3.9/site-packages/pcluster/cli/entrypoint.py", line 197, in _run_operation
    return args.func(args, extra_args)
  File "/mnt/c/Users/t_kono/Develop/pcluster-test/env/lib/python3.9/site-packages/pcluster/cli/commands/configure/command.py", line 38, in execute
    configure(args)
  File "/mnt/c/Users/t_kono/Develop/pcluster-test/env/lib/python3.9/site-packages/pcluster/cli/commands/configure/easyconfig.py", line 280, in configure
    vpc_parameters = _create_vpc_parameters(scheduler, head_node_instance_type, compute_instance_types, cluster_size)
  File "/mnt/c/Users/t_kono/Develop/pcluster-test/env/lib/python3.9/site-packages/pcluster/cli/commands/configure/easyconfig.py", line 336, in _create_vpc_parameters
    vpc_and_subnets = _get_vpcs_and_subnets()
  File "/mnt/c/Users/t_kono/Develop/pcluster-test/env/lib/python3.9/site-packages/pcluster/cli/commands/configure/utils.py", line 31, in wrapper
    return func(*args, **kwargs)
  File "/mnt/c/Users/t_kono/Develop/pcluster-test/env/lib/python3.9/site-packages/pcluster/cli/commands/configure/easyconfig.py", line 92, in _get_vpcs_and_subnets
    subnets = _get_subnets(ec2_client, vpc_id)
  File "/mnt/c/Users/t_kono/Develop/pcluster-test/env/lib/python3.9/site-packages/pcluster/cli/commands/configure/easyconfig.py", line 110, in _get_subnets
    ("size", _extract_subnet_size(subnet.get("CidrBlock"))),
  File "/mnt/c/Users/t_kono/Develop/pcluster-test/env/lib/python3.9/site-packages/pcluster/cli/commands/configure/easyconfig.py", line 71, in _extract_subnet_size
    return 2 ** (32 - int(cidr.split("/")[1]))
AttributeError: 'NoneType' object has no attribute 'split'
```
and when the subnet is IPv6 CIDR only subnet, ```subnet.get("CidrBlock")``` is empty because the API doesn't return that. So existing IPv6 CIDR only subnet occurs ```pcluster configure``` command failure.

https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeSubnets.html
https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Subnet.html
> cidrBlock
The IPv4 CIDR block assigned to the subnet.
Type: String
Required: No

**Comment by k11o** [comment]
Sorry I checked issues but didn't in pull requests. This is duplicate to this (https://github.com/aws/aws-parallelcluster/pull/4567).

**Comment by hanwen-pcluste** [comment]
I merged https://github.com/aws/aws-parallelcluster/pull/4567. The fix will be included in ParallelCluster 3.5.0

**Comment by maurizp** [comment]
Parallel Cluster 3.5.0 has been released and includes this patch.
